packages/scipy/patches/0016-Drop-unsupported-array-constructors-in-ARPACK.patch

```diff
@@ -0,0 +1,1255 @@
+ .../linalg/_eigen/arpack/ARPACK/SRC/cnapps.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/cnaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/cnaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/cneupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/cngets.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dgetv0.f     |  6 +++---
+ .../linalg/_eigen/arpack/ARPACK/SRC/dnaitr.f     |  8 ++++----
+ .../linalg/_eigen/arpack/ARPACK/SRC/dnapps.f     | 16 ++++++++--------
+ .../linalg/_eigen/arpack/ARPACK/SRC/dnaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/dnaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dneupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dngets.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dsaitr.f     |  8 ++++----
+ .../linalg/_eigen/arpack/ARPACK/SRC/dsapps.f     |  6 +++---
+ .../linalg/_eigen/arpack/ARPACK/SRC/dsaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/dsaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dseupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/dsgets.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/sgetv0.f     |  6 +++---
+ .../linalg/_eigen/arpack/ARPACK/SRC/snaitr.f     |  8 ++++----
+ .../linalg/_eigen/arpack/ARPACK/SRC/snapps.f     | 16 ++++++++--------
+ .../linalg/_eigen/arpack/ARPACK/SRC/snaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/snaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/sneupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/sngets.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/ssaitr.f     |  8 ++++----
+ .../linalg/_eigen/arpack/ARPACK/SRC/ssapps.f     |  6 +++---
+ .../linalg/_eigen/arpack/ARPACK/SRC/ssaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/ssaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/sseupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/ssgets.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/zgetv0.f     |  6 +++---
+ .../linalg/_eigen/arpack/ARPACK/SRC/znaitr.f     |  8 ++++----
+ .../linalg/_eigen/arpack/ARPACK/SRC/znapps.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/znaup2.f     | 14 +++++++-------
+ .../linalg/_eigen/arpack/ARPACK/SRC/znaupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/zneupd.f     |  4 ++--
+ .../linalg/_eigen/arpack/ARPACK/SRC/zngets.f     |  4 ++--
+ 40 files changed, 150 insertions(+), 150 deletions(-)
+
+diff --git a/scipy/sparse/linalg/_eigen/arpack/ARPACK/SRC/cgetv0.f b/scipy/sparse/linalg/_eigen/arpack/ARPACK/SRC/cgetv0.f
+index c231eadcb..fc4b24b92 100644
+--- a/scipy/sparse/linalg/_eigen/arpack/ARPACK/SRC/cgetv0.f
++++ b/scipy/sparse/linalg/_eigen/arpack/ARPACK/SRC/cgetv0.f
+@@ -361,9 +361,9 @@ c     | Check for further orthogonalization. |
+ c     %--------------------------------------%
+ c
+       if (msglvl .gt. 2) then
+-          call svout (logfil, 1, [rnorm0], ndigit,
```

Comment:
Rather than applying a huge repetitive patch I'd prefer to fix this in `_f2c_fixes` or in f2c itself.